pkg/agent/cniserver/server.go

```diff
@@ -521,6 +552,49 @@ func (s *CNIServer) Run(stopCh <-chan struct{}) {
 	<-stopCh
 }
 
+// interceptAdd handles Add request in policy only mode. Another CNI must already
+// be called prior to Antrea CNI to allocate IP and ports. Antrea takes allocated port
+// and hooks it to OVS br-int.
+func (s *CNIServer) interceptAdd(cniConfig *CNIConfig) (*cnipb.CniCmdResponse, error) {
+	klog.Infof("CNI Chaining: add")
+	prevResult, response := s.parsePrevResultFromRequest(cniConfig.NetworkConfig)
+	if response != nil {
+		klog.Infof("Failed to parse prev result for container %s.", cniConfig.ContainerId)
```

Comment:
nit: remove period from the end of msg in Infof

Reply:
done